mitodl/hugo-course-publisher
# some tooling tweaks

#### Pre-Flight checklist


#### What are the relevant tickets?

I basically did some tweaks to simplify things a little bit and to get the build passing on travis.

#### What's this PR do?

- moves code out of `app` directory
- reconfigures `Dockerfile` and `docker-compose.yml` to deal with that change
- remove manual `install_yarn` script, in favor of using a) built-in yarn in docker `node` container and installing via the [recommended method](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#using-yarn) on travis
- move webpack config files into `src/webpack`
- add an `.eslintrc` so that linting works (I just added the most basic one possible)
- ran `prettier` on the repo

I think that was about it.

#### How should this be manually tested?

- build should pass on travis
- `docker-compose build` should work without erroring
- the app should run on `docker-compose up` as it did before
